main.py

```diff
@@ -714,6 +714,9 @@ def ui_access_wrapper(self, *args, **kwargs):
         r'/explorationdataextractionhandler', admin.DataExtractionQueryHandler),
     get_redirect_route(
         r'/sendDummyMailToAdminHandler', admin.SendDummyMailToAdminHandler),
+    get_redirect_route(
+        r'/interactionsByExplorationId',
+        admin.InteractionsByExplorationId),
```

Comment:
Can we move this to the line above? (max 80 chars in a line)

Reply:
done